src/openms/source/ANALYSIS/QUANTITATION/PeptideAndProteinQuant.cpp

```diff
@@ -48,13 +49,17 @@ namespace OpenMS
     DefaultParamHandler("PeptideAndProteinQuant"), stats_(), pep_quant_(),
     prot_quant_()
   {
+    std::vector<std::string> true_false = {"true","false"};
+
+    defaults_.setValue("method", "TOP3", "- TOP3 - quantify based on three most abundant peptides (number can be changed in 'top').\n- iBAQ (intensity based absolute quantification), will override top with 0 and average with sum.");
```

Comment:
I think using TOP3 as parameter name could be confusing. "top" would be better I think.